### DISCUSSION TITLE
Portfolio graph

### DISCUSSION THREAD

**Discussion by Akash1yadavv** [community/community#36297] [Other Feature Feedback, Questions, & Ideas]
I have completed more than 100 commits on github but I able to see only 10 to 15 commits on my profile, Why??

**Comment by radar-blip** [comment]
The commits made in private repositories wont show. You must enable them in settings, like below:

![profiles](https://user-images.githubusercontent.com/39196767/196025094-d1589a91-ab41-499c-bc04-9c35658f36fd.png)

Also, here it shows you made 11 commits, but that is only for the month of October

![Screenshot from 2022-10-16 04-04-46](https://user-images.githubusercontent.com/39196767/196025261-200740d3-2080-455b-8307-79ce78871ebd.png)

Everything else looks fine.

**Reply by Akash1yadavv** [reply]
But how to include private contribution ??

**Reply by DSDmark** [reply]
Just click on that link and scroll down, you can see **contributions and activity** section

[**Click Me**](https://github.com/settings/profile)

**Reply by AbdulAhad365** [reply]
Just remain determined and improve your coding. If still same change your methord of creating the logic.

**Comment by ossanna16** [comment]
Hey there @Akash1yadavv and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by Akash1yadavv** [reply]
😊

**Reply by DSDmark** [reply]
Hey @Akash1yadavv,
if you got your answer, then you should to close this question by click on the accept answer button.
